src/platforms/esp32/components/avm_builtins/gpio_driver.c

```diff
@@ -171,7 +194,17 @@ static inline term set_pin_pull_mode(Context *ctx, term gpio_num_term, term pull
 
 static inline term hold_en(term gpio_num_term)
 {
-    int gpio_num = term_to_int(gpio_num_term);
+    gpio_num_t gpio_num;
+    if (LIKELY(term_is_integer(gpio_num_term))) {
+        avm_int_t pin_int =  term_to_int32(gpio_num_term);
```

Comment:
formatting issue here.
if you want to format just a small portion of the file, you can also use `clang-format -i --lines=197:207 src/platforms/esp32/components/avm_builtins/gpio_driver.c`.
When doing it just on small chunks I also go for `ColumnLimit: 100` setting in `.clang-format`, sadly that setting is not always reliable and with structs it messes up things.

Reply:
Thank you. That tip helps a lot. Sometimes I hesitate to run clang format over the files because it will make extra undesirable changes to code outside the lines changed in the PR. I didn't realize there was a way to check and fix lines selectively.